daq/faucet_states_collector.py

```diff
@@ -49,12 +49,12 @@ def wrapped(self, *args, **kwargs):
 TOPOLOGY_LINK_MAP = "physical_stack_links"
 TOPOLOGY_LACP = "lacp_lag_status"
 TOPOLOGY_ROOT = "active_root"
-
+RAW_CONFIG = "raw_config"
```

Comment:
likely should be qualified as dps_config, since there are other kinds of config.  It's a more interesting qualifier than "raw"

Reply:
Resolved. PTAL